cards/card_component.js

```diff
@@ -9,6 +9,11 @@ BaseCard["{{componentName}}"] = class extends ANSWERS.Component {
 
     this.verticalKey = data.verticalKey;
     this.result = data.result || {};
+
+    this.globalOptions = this.analyticsReporter && this.analyticsReporter._globalOptions;
```

Comment:
nit: can we use optional chaining here? That would let us shorten these lines. Here's an example: `this.globalOptions = this.analyticsReporter?._globalOptions;`